src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

Необходимо явно указывать свойства, которые будут анимированы, иначе используется transition: all
transition: all достаточно требователен к ресурсам браузера и на старых компьютерах может ухудшать пользовательский опыт
```suggestion
    transition: transform 300ms;
```